tree/ntuple/v7/inc/ROOT/RNTupleDescriptor.hxx

```diff
@@ -181,10 +183,11 @@ public:
    DescriptorId_t GetFieldId() const { return fFieldId; }
    std::uint32_t GetIndex() const { return fIndex; }
    std::uint16_t GetRepresentationIndex() const { return fRepresentationIndex; }
-   std::uint64_t GetFirstElementIndex() const { return fFirstElementIndex; }
+   std::uint64_t GetFirstElementIndex() const { return std::abs(fFirstElementIndex); }
```

Comment:
Do we expect to call this function for suppressed columns? Maybe add an assertion of not?

Reply:
Yes, this is called also for suppressed columns. E.g. in the RNTupleSerializer.